python/cudf/cudf/core/indexed_frame.py

```diff
@@ -270,6 +271,10 @@ def _from_data(
         out._index = RangeIndex(out._data.nrows) if index is None else index
         return out
 
+    @_cudf_nvtx_annotate
```

Comment:
```suggestion
    @classmethod
    @_cudf_nvtx_annotate
```

Reply:
Looks like `self` is used correctly here, must have just had a typo or copy-pasted differently in the other case.